README.md

```diff
@@ -190,6 +190,13 @@ config :mime, :types, %{
 }
 ```
 
+And then re-compile mime: (per: https://hexdocs.pm/mime/MIME.html)
+
+```shell
+mix deps.clean mime --build
+mix deps.get
```

Comment:
@alanpeabody Please recheck commands (and also for `plug < 1.2`). I think actual compilation happens on first app run (on mix test for MIX_ENV=test).